### PR TITLE
fix initial placement and make game more exciting

### PR DESCRIPTION
**Pull request by renancaraujo**

### CODE REVIEW

File: lib/game/game_config.dart [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   static double invertDistance = 700.0;
   static double maxBlinkCount = 3.0;
   static double maxClouds = 6.0;
   static double maxObstacleLength = 3.0;
   static int maxObstacleDuplication = 2;
-  static double maxSpeed = 13.0;
+  static double maxSpeed = 50.0;
   static double minJumpHeight = 35.0;
-  static double speed = 6.5;
+  static double speed = 12;
```
Comment (luanpotter) [inline lib/game/game_config.dart:19]: wow that is A LOT more interesting haha

File: lib/game/t_rex/t_rex.dart [hunk 2]
```diff
@@ -52,56 +55,71 @@ class TRex extends PositionComponent
...
     // intro related
     if (jumpCount == 1 && !playingIntro && !hasPlayedIntro) {
       status = TRexStatus.intro;
     }
     if (playingIntro && x < TRexConfig.startXPos) {
-      x += ((TRexConfig.startXPos / TRexConfig.introDuration) * t * 5000);
+      x += (TRexConfig.startXPos / TRexConfig.introDuration) * t * 5000;
+    }
+
+    if(status == TRexStatus.crashed) {
```
Comment (luanpotter) [inline lib/game/t_rex/t_rex.dart:104]: `if (`

File: lib/game/t_rex/t_rex.dart [hunk 3]
```diff
@@ -52,56 +55,71 @@ class TRex extends PositionComponent
...
+      actualDino.y != 405.0 ? print(actualDino.y) : 0.0;
```
Comment (luanpotter) [inline lib/game/t_rex/t_rex.dart:105]: what is this log?? haha

Comment (renancaraujo) [reply lib/game/t_rex/t_rex.dart:105]: uhehehuehuhe

File: lib/game/t_rex/t_rex.dart (hunk 2 above)
Comment (renancaraujo) [reply lib/game/t_rex/t_rex.dart:104]: no

**Pull request merged**